### PR TITLE
fix(security): json schema and json5

### PR DESCRIPTION
**Pull request by benjlevesque**
https://github.com/RequestNetwork/requestNetwork/security/dependabot/51
https://github.com/RequestNetwork/requestNetwork/security/dependabot/165

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,12 +53,16 @@
...
+    "semver": "^7.5.4",
+    "json-schema": "^0.4.0",
+    "json5": "^2.2.1"
   },
   "resolutionsDocs": {
     "underscore": "https://github.com/RequestNetwork/requestNetwork/security/dependabot/14",
     "crypto-js": "https://github.com/RequestNetwork/requestNetwork/security/dependabot/207",
     "minimist": "https://github.com/RequestNetwork/requestNetwork/security/dependabot/177",
-    "semver": "https://github.com/RequestNetwork/requestNetwork/security/dependabot/197"
+    "semver": "https://github.com/RequestNetwork/requestNetwork/security/dependabot/197",
```
Comment (yomarion) [inline package.json:64]: I can't find any doc for "resolutionsDocs" @benjlevesque , how does it work?

**Review by yomarion — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (benjlevesque) [reply package.json:64]: it does not 😇

Comment (benjlevesque) [reply package.json:64]: it's a pattern I found to document package.json...

Comment (benjlevesque) [reply package.json:64]: since it's not json5 and does not support comments

**Review by alexandre-abrioux — APPROVED** [review]

**Pull request merged**